docs/_interface/peer-to-peer.md

```diff
@@ -118,67 +117,72 @@ To check the ports used by geth and also find your enode URI run:
 
 ## Custom Networks
 
-Sometimes you might not need to connect to the live public network, you can instead choose
-to create your own private testnet. This is very useful if you don't need to test external
-contracts and want just to test the technology, because you won't have to compete with
-other miners and will easily generate a lot of test ether to play around (replace 12345
-with any non-negative number):
+It is often useful for developers to connect to private test networks rather than public testnets or Etheruem mainnet. These sandbox environments allow block creation without competing against other miners, easy minting of test ether and give freedom to break things without real-world consequences. A private network is started by providing a value to `--networkid` that is not used by any other existing public network ([Chainlist](https://chainlist.org)) and creating a custom `genesis.json` file. Detailed instructions for this are available on the [Private Networks page](/docs/interface/private-network).
 
-	geth -—networkid="12345" console
+```shell
 
-It is also possible to run geth with a custom genesis block from a JSON file by supplying
-the `geth init /path/to/file` command. The genesis JSON file should have the following format:
+geth --networkid="12345" genesis.json
+
+```
 
-```js
-{
-  "alloc": {
-    "dbdbdb2cbd23b783741e8d7fcf51e459b497e4a6": { 
-        "balance": "1606938044258990275541962092341162602522202993782792835301376"
-    },
-    "e6716f9544a56c530d868e4bfbacb172315bdead": {
-        "balance": "1606938044258990275541962092341162602522202993782792835301376"
-    },
-    ...
-  },
-  "nonce": "0x000000000000002a",
-  "difficulty": "0x020000",
-  "mixhash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-  "coinbase": "0x0000000000000000000000000000000000000000",
-  "timestamp": "0x00",
-  "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-  "extraData": "0x",
-  "gasLimit": "0x2fefd8"
-}
-``` 
 
 ## Static nodes
 
-Geth also supports a feature called static nodes if you have certain peers you always want
-to connect to. Static nodes are re-connected on disconnects. You can configure permanent
-static nodes by putting something like the following into
-`<datadir>/geth/static-nodes.json`:
+Geth also supports static nodes. Static nodes are specific peers that are always connected to. Geth reconnects to these peers automatically when it is restarted. Specific nodes are defined to be statc nodes by saving their enode addresses to a json file and providing the file path to Gteh at startup:
+
+```shell
 
-```js
+geth <other flags> datadir/geth/static-nodes.json
```

Comment:
hm does this work? I thought this file has to be stored in `<datadir>/geth/static-nodes.json` and then geth will automatically find it without the user having to pass it as argument.

Reply:
yes you are right. Thanks for the catch - updating now.